operate/client/e2e-playwright/tests/processInstanceMigration.spec.ts

```diff
@@ -799,14 +799,37 @@ test.describe.serial('Process Instance Migration', () => {
     /**
      * Expect that the escalation boundary event has been migrated.
      * In v1 and v2 of the process the escalation boundary event has a different escalation code (123).
-     * Since the escalation code is static, it is expected that the code in the target process (456) is used.
+     * Since the escalation code is static, it is expected that the code in the target process (234) is used.
      *
-     * To test this, an escalation throw event with escalation code (456) is triggered. When the escalation
+     * To test this, an escalation throw event with escalation code (234) is triggered. When the escalation
      * boundary event becomes selectable, it means it has caught the escalation correctly.
      */
     await expect(
-      await processInstancePage.diagram.getLabeledElement('Escalation event'),
+      await processInstancePage.diagram.getLabeledElement(
+        'Escalation boundary event',
```

Comment:
I changed the element name, because it also found "Escalation event sub process" here.